src/core/parser_internal/ignore_worker.hpp

```diff
@@ -0,0 +1,40 @@
+#pragma once
+
+#include "docgen_worker.hpp"
+
+namespace docgen {
+namespace core {
+namespace parser_internal {
+
+struct IgnoreWorker_routines : private routine_details_t
+{
+	protected:
+		static constexpr const routine_t on_start_ = [](worker_t *, const tok_t&, dest_t& writer) {
+			writer.stop_writing();
+		};
+		static constexpr const routine_t on_start_recurse_ = [](worker_t *worker, const tok_t& t, dest_t& d) {
+			on_start_(worker, t, d);
+			worker->inject_worker(ParseWorker(*worker).reset());
+		};
+		static constexpr const routine_t on_stop_ = [](worker_t *, const tok_t&, dest_t& writer) {
+			writer.start_writing();
+		};
+};
+
+class IgnoreWorker : public worker_class_t, private IgnoreWorker_routines
+{
+	public:
+		IgnoreWorker(token_t from, token_t until)
```

Comment:
copying tokens is unnecessary since we're holding `std::string`. use `const token_t&`.

Reply:
Whoops, this was a holdover from when I was using symbols here. Fixed.